handlers/main.yml

```diff
@@ -4,4 +4,3 @@
     name: "jira"
     state: restarted
     daemon_reload: true
-    enabled: true
```

Comment:
I've tested your modifications but I get the same error when I first install jira, it can't do the enable until the service is up.
So I added a service stop/start task in the handlers and tested it on the Creuset Server